File: typescript/Basic/ApiKey/src/apikey.ts
import { ApiClient, FrequenciesApi, CurrenciesApi } from '@factset/sdk-paengine';
import { ConfidentialClient } from '@factset/sdk-utils';

async function apikey() {
    const client = new ApiClient();
    
   const factSetApiKey = client.authentications['FactSetApiKey'];
   factSetApiKey.username = 'USERNAME-SERIAL';
   factSetApiKey.password = 'API-KEY';

    const frequenciesApi = new FrequenciesApi(client);
    const result = await frequenciesApi.getPAFrequencies();
   
    console.log(result.data);
}

apikey();

File: python/Basic/ApiKey/apikey.py
from fds.sdk.PAEngine import ApiClient, ApiException, Configuration
from fds.sdk.PAEngine.api.frequencies_api import FrequenciesApi

conf = Configuration(
    username='USERNAME-SERIAL',
    password='API-KEY'
)

with ApiClient(conf) as api_client:
    frequenciesApi = FrequenciesApi(api_client)

    try:
        frequencyRoot = frequenciesApi.get_pa_frequencies()
        print(frequencyRoot.data)
    except ApiException as e:
        print("Error calling API: %s\n" % e)
